aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -54,6 +70,25 @@ else
 	curl --silent -H "Accept: application/json" -H "Content-type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d '{"name": "Ethos", "type": "security.profile", "description": "Ethos Default RunTime Profile", "encrypt_all_envs": true}' $WEB_URL/securityprofiles
 fi
 
+EXISTING_ARTIFACTORY=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/registries/artifactory-test)
+
+if [[ "$EXISTING_ARTIFACTORY" == "200" ]]; then
+	log "Artifactory Integration exists..."
+else
+	curl --silent -H "Accept: application/json" -H "Content-type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d  '{"prefixes": [], "auto_pull": false, "auto_pull_time": "03:00", "auto_pull_max": 100, "name": "artifactory-test", "type": "ART", "webhook": {"enabled": false, "url": "", "auth_token": ""}, "url": "$Artifactory_URL", "username": "$Artifactory_USERNAME", "password": "$Artifactory_PASSWORD"}' $WEB_URL/registries
```

Comment:
Let's stick with capital letters for the env variables: `$ARTIFACTORY_URL`, `$ARTIFACTORY_USERNAME`, etc.